src/components/asthma/views/AsthmaLibraryCategoryView/AsthmaLibraryCategoryView.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Layout, NavigationBar, ResourceImageAlignment, Title } from '../../../presentational';
```

Comment:
Can be decoupled from Asthma

Reply:
Good idea.  Moved to the common components and renamed to `ResourceListView`.